CMakeLists.txt

```diff
@@ -104,10 +105,12 @@ endmacro()
 #
 set(URIPARSER_EXTRA_COMPILE_FLAGS)
 
+if(HIDE_SYMBOLS)
 check_c_compiler_flag("-fvisibility=hidden" URIPARSER_COMPILER_SUPPORTS_VISIBILITY)
 if(URIPARSER_COMPILER_SUPPORTS_VISIBILITY)
     set(URIPARSER_EXTRA_COMPILE_FLAGS "${URIPARSER_EXTRA_COMPILE_FLAGS} -fvisibility=hidden")
 endif()
+endif(HIDE_SYMBOLS)
```

Comment:
Please use plain `endif()` for consistency.

Also please increase indent of the new if's body by 4 spaces.